var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -324,6 +324,10 @@ class Llvm(CMakePackage):
     conflicts('+clang_extra', when='~clang')
     conflicts('+lldb',        when='~clang')
 
+    # Github issue #4986
+    patch('llvm_gcc7.patch', when='@4.0.0+lldb ^gcc@7.0:')
+    patch('llvm_gcc7.patch', when='@4.0.1+lldb ^gcc@7.0:')
```

Comment:
You can combine these:
```python
patch('llvm_gcc7.patch', when='@4.0.0:4.0.1+lldb ^gcc@7.0:')
```

Reply:
Yeah, I was wondering about it. Looks pretty interesting as it will combine two ranges...